### ISSUE TITLE
Add new plugin

### ISSUE THREAD

**Issue by pazicb** [JiHong88/SunEditor#131]
Hi,
can you help me please? I created the plugin fileManager.js.
I loaded it using <script src = "fileManager.js"> </script>.
I defined plugins: [fileManager].
I added the plugin to the buttonList.
Everything works correctly, but unfortunately, other plugins such as link, image, video, etc. have stopped working.
How do I make all plugins work?

Thank You

**Comment by JiHong88** [comment]
The SUNEDITOR object was created using the [init](https://github.com/JiHong88/SunEditor#init-function) function.
```
window.SUNEDITOR = suneditor.init({
    plugins: plugins
});
```
Overriding the plugins option seems to have removed other plugins.

The immediate way is to define all the plugins using webpack.
```
plugins: [fileManager, font, fontSize, link, image, ...]
```
I will update tomorrow by modifying the create function so that the plugin is not removed.

**Comment by pazicb** [comment]
Hi
thank you for your reply. Unfortunately, font, fontSize, link and image etc. are not defined.

**Comment by JiHong88** [comment]
Hi, @pazicb 

Fixed in [v2.20.0](https://github.com/JiHong88/SunEditor/releases/tag/2.20.0)

Thank you.
